CONTRIBUTING.md

```diff
@@ -0,0 +1,193 @@
+# Contributing to Numberscope
+
+## If you are new to software development, watch the video tutorials
+
+Watch the video series on contributing to Numberscope
+[here](https://www.youtube.com/playlist?list=PLA4KIQBQQRb5ccOdr9v0iLw_fKHup1PkU).
+This video series will introduce you to Numberscope, GitHub, the software we
+use in the Numberscope project, and the workflow for contributing to
+Numberscope.
+
+## If you have software dev experience, here's the TL;DR
+
```

Comment:
Maybe a title like "Here's the TL;DR / overview" because this is useful even if you don't have dev experience.  It will give the big picture.